react-native-fbsdk.podspec

```diff
@@ -10,21 +10,21 @@ Pod::Spec.new do |s|
   s.license       = package['license']
   s.homepage      = package['homepage']
   s.source        = { :git => 'https://github.com/facebook/react-native-fbsdk.git', :tag => "v#{package['version']}" }
-  s.platform      = :ios, '7.0'
+  s.platform      = :ios, '8.0'
```

Comment:
this is not needed, but i thought it makes sense because the deployment target of the fb sdks is set to `8.0`